crates/nu-cli/src/commands/math/median.rs

```diff
@@ -61,24 +61,25 @@ enum Pick {
 }
 
 pub fn median(values: &[Value], name: &Tag) -> Result<Value, ShellError> {
-    let take = if values.len() % 2 == 0 {
-        Pick::MedianAverage
-    } else {
-        Pick::Median
-    };
-
     let mut sorted = vec![];
 
     for item in values {
         sorted.push(item.clone());
     }
 
     crate::commands::sort_by::sort(&mut sorted, &[], name)?;
+    sorted.dedup();
```

Comment:
Does this mean that if some values are the same in the list, they will be removed and not considered when retrieving the median?